<?xml version="1.0" encoding="UTF-8"?>
<robot generated="20141128 14:45:07.114" generator="Robot 2.8.4 (Python 2.7.6 on linux2)">
<suite source="/home/supersherlock/Desktop/CSC-183-Activities/BankFinal/BankFinalRobotTest/test/robot" id="s1" name="Robot">
<suite source="/home/supersherlock/Desktop/CSC-183-Activities/BankFinal/BankFinalRobotTest/test/robot/tests" id="s1-s1" name="Tests">
<suite source="/home/supersherlock/Desktop/CSC-183-Activities/BankFinal/BankFinalRobotTest/test/robot/tests/bank.txt" id="s1-s1-s1" name="Bank">
<test id="s1-s1-s1-t1" name="Retrieve customer balance">
<kw type="kw" name="steps.Given I create the account 1111 with balance 50">
<doc></doc>
<arguments>
</arguments>
<kw type="kw" name="Create Account ${account_number} ${balance}">
<doc></doc>
<arguments>
</arguments>
<msg timestamp="20141128 14:45:07.157" level="FAIL">No keyword with name 'Create Account ${account_number} ${balance}' found.</msg>
<status status="FAIL" endtime="20141128 14:45:07.157" starttime="20141128 14:45:07.157"></status>
</kw>
<status status="FAIL" endtime="20141128 14:45:07.157" starttime="20141128 14:45:07.156"></status>
</kw>
<doc></doc>
<tags>
</tags>
<status status="FAIL" endtime="20141128 14:45:07.158" critical="yes" starttime="20141128 14:45:07.155">No keyword with name 'Create Account ${account_number} ${balance}' found.</status>
</test>
<doc>Example Robot tests for
Bank application</doc>
<metadata>
</metadata>
<status status="FAIL" endtime="20141128 14:45:07.159" starttime="20141128 14:45:07.146"></status>
</suite>
<doc></doc>
<metadata>
</metadata>
<status status="FAIL" endtime="20141128 14:45:07.159" starttime="20141128 14:45:07.144"></status>
</suite>
<doc></doc>
<metadata>
</metadata>
<status status="FAIL" endtime="20141128 14:45:07.160" starttime="20141128 14:45:07.115"></status>
</suite>
<statistics>
<total>
<stat fail="1" pass="0">Critical Tests</stat>
<stat fail="1" pass="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat fail="1" pass="0" id="s1" name="Robot">Robot</stat>
<stat fail="1" pass="0" id="s1-s1" name="Tests">Robot.Tests</stat>
<stat fail="1" pass="0" id="s1-s1-s1" name="Bank">Robot.Tests.Bank</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20141128 14:45:07.150" level="ERROR">Error in file '/home/supersherlock/Desktop/CSC-183-Activities/BankFinal/BankFinalRobotTest/test/robot/steps/resource.txt': Importing test library '/home/supersherlock/Desktop/CSC-183-Activities/BankFinal/BankFinalRobotTest/test/robot/steps/robotLibrary.py' failed: SyntaxError: Non-ASCII character '\xef' in file /home/supersherlock/Desktop/CSC-183-Activities/BankFinal/BankFinalRobotTest/test/robot/steps/robotLibrary.py on line 28, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (robotLibrary.py, line 28)
Traceback (most recent call last):
  None
PYTHONPATH:
  /usr/local/lib/python2.7/dist-packages/robot/libraries
  /usr/local/lib/python2.7/dist-packages
  /usr/local/bin
  /usr/local/lib/python2.7/dist-packages/pip-1.5.6-py2.7.egg
  /usr/lib/python2.7/dist-packages
  /usr/lib/python2.7
  /usr/lib/python2.7/plat-x86_64-linux-gnu
  /usr/lib/python2.7/lib-tk
  /usr/lib/python2.7/lib-old
  /usr/lib/python2.7/lib-dynload
  /usr/lib/python2.7/dist-packages/PILcompat
  /usr/lib/python2.7/dist-packages/gst-0.10
  /usr/lib/python2.7/dist-packages/gtk-2.0
  /usr/lib/pymodules/python2.7
  /usr/lib/python2.7/dist-packages/ubuntu-sso-client
  /usr/lib/python2.7/dist-packages/wx-2.8-gtk2-unicode
  .
  /home/supersherlock/Desktop/CSC-183-Activities/BankFinal/BankFinalRobotTest</msg>
</errors>
</robot>
